yarn-error.log

```diff
@@ -0,0 +1,36 @@
+Arguments: 
```

Comment:
This file is required? Seems like auto-generated we can delete it

Reply:
Deleted the file